src/UI/Internal/SideBar.elm

```diff
@@ -1,6 +1,20 @@
 module UI.Internal.SideBar exposing (desktopColumn, mobileDrawer)
 
-import Element exposing (Attribute, Element, fill, fillPortion, height, padding, paddingEach, paddingXY, px, scrollbarY, shrink, spacing, width)
+import Element
+    exposing
+        ( Attribute
+        , Element
+        , fill
+        , fillPortion
+        , height
+        , padding
```

Comment:
Can we add a rule so that we can have `(Element.width fill)` and `(Element.paddingXY 0 12)` but not `(width Element.fill)` neither `(paddingXY 0 12)` ?? (Import exposing right elements like fill/rgb/shrink/px but never left ones like height/width/padding)

Reply:
Yeah, I think we will have to write our custom rules to support these use cases.